tools/.gitignore

```diff
@@ -0,0 +1,36 @@
+*.class
```

Comment:
Can't we use the parent .gitignore file?

Reply:
removed .gitignore file